lib/util/src/hse_err.c

```diff
@@ -13,11 +13,10 @@
 
 #include <bsd/string.h>
 
-char hse_merr_bug0[] _merr_attributes = "hse_merr_bug0u";
-char hse_merr_bug1[] _merr_attributes = "hse_merr_bug1u";
-char hse_merr_bug2[] _merr_attributes = "hse_merr_bug2u";
-char hse_merr_bug3[] _merr_attributes = "hse_merr_bug3u";
-char hse_merr_base[] _merr_attributes = "hse_merr_baseu";
+char invalid_file_bug0[] _merr_attributes = "invalid_file_bug0";
+char invalid_file_bug1[] _merr_attributes = "invalid_file_bug1";
+char invalid_file_bug2[] _merr_attributes = "invalid_file_bug2";
+char hse_merr_base[] _merr_attributes = "hse_merr_base";
```

Comment:
Maybe just call this merr_base, and the section merr_base. Will let @beckerg make the final decisions.

Reply:
That might be confusing, and has the same grep problem when searching for "hse_merr" in the object dump.  What I'd like to do is get rid of hse_merr_base entirely, but that's a different patch, lemme create a ticket for that...

Reply:
Will let the new ticket address this.